test/cluster.sh

```diff
@@ -69,6 +69,7 @@ function install_knative_serving() {
   # TODO: Should we install build from a nightly release?
   # The latest released Build is always at this location.
   INSTALL_BUILD_YAML=https://storage.googleapis.com/knative-releases/build/latest/release.yaml
+  INSTALL_PIPELINE_YAML=https://storage.googleapis.com/knative-nightly/build-pipeline/latest/release.yaml
```

Comment:
I actually don't like either of these lines, but Pipelines a lot less because it means a broken nightly can break serving (at least Build it's the last release, which is problematic for release branches).

I'd much prefer if these uses the checked in yaml.

Reply:
@mattmoor yep, makes sense to use a checked yaml instead :angel: 
I'll update :wink: